fungible-token/src/lib.rs

```diff
@@ -61,8 +77,20 @@ impl FungibleToken {
     fn get_balance(&self, account: &ActorId) -> u128 {
         *self.balances.get(account).unwrap_or(&0)
     }
+    /// Executed on receiving `fungible-token-message::BalanceOf`, returns token balance of `account`.
+    fn balance_of(&self, account: &ActorId) -> u128 {
+        self.get_balance(account)
+    }
+    /// Executed on receiving `fungible-token-messages::MintInput`.
+    /// If executed by creator or admins of the token then mints `amount` tokens into `account`.
     fn mint(&mut self, account: &ActorId, amount: u128) {
-        let zero = ActorId::new(H256::zero().to_fixed_bytes());
+        unsafe {
+            let source = msg::source();
+            if FUNGIBLE_TOKEN.creator != source && !FUNGIBLE_TOKEN.admins.contains(&source) {
+                panic!("FungibleToken: Only token creator or designated admins can mint tokens.");
+            }
+        }
+        let zero = ActorId::new([0u8; 32]);
```

Comment:
May be it will be convenient to declare a constant for the zero address? There are a lot of places where checks for a zero address are used